pkg/function/restore_data_using_kopia_server.go

```diff
@@ -0,0 +1,261 @@
+package function
```

Comment:
Please add LICENSE headers to the newly created files

Reply:
Done !